php-classes/Slate/Progress/Narratives/Report.php

```diff
@@ -71,4 +64,100 @@ class Report extends \VersionedRecord
             'class' => \Slate\Term::class
         ]
     ];
-}
\ No newline at end of file
+
+    public static $searchConditions = [
+        'narrativeID' => [
```

Comment:
@BillClinton these are typically keyed after the field they search, capitalizing them like `narrativeID` isn't consistent with anywhere else, so change that to `NarrativeID`

For the qualifiers the pattern is to use hyphens as word separator like good does, no `narrativeid:123` -> `narrative-id:123`